### PR TITLE
Housekeeping, and other jazz

### PR DESCRIPTION
**Pull request by evettetelyas**
lots of testing updates
other things of note:
- deleted a bunch of dead code i didn't really understand WRT sending emails, but it still works in dev so thats cool
- added a partial for the merchant dashboard and split the route for admin/merchant/id from merchant/dashboard#index to admin/merchant#index because it should not be accessible to an admin from /merchant route. (see finding below regarding the routes)
- had to add a tidbit to rails_helper but i would just test this locally before merging and make sure its okay
- changed user.works_here from @current_merchant to self.
- a bunch of attempts at testing email delivery but nooooooooOOOOOooOOoO
- also a note on this route:
```
  get '/admin/users/:user_id/orders/:order_id', to: 'orders#show'
```

i didnt have it locally but it was in master, so i kept it, and will look into what it does (maybe i accidentally deleted it before committing or something?)

and lastly the note from bullet 2- 
i am not able to push this up after the fact somehow BUT; merchants/dashboard#order_show needs this now due to the new routes for admin to view /merchant/orders:
```
  def order_show
    if current_admin?
      @order = Order.find(params[:id])
      @merchant_items = @order.merchant_items(Merchant.find(session[:merchant_id]))
    else
      @order = Order.find(params[:id])
      user = User.find(session[:user_id])
      @merchant_items = @order.merchant_items(user.merchant)
    end
  end
```
^ this is is because the params are now different for both admins and merchants reaching the order show page

### CODE REVIEW

File: spec/rails_helper.rb [hunk 1]
```diff
@@ -36,6 +36,7 @@
 RSpec.configure do |config|
   # Remove this line if you're not using ActiveRecord or ActiveRecord fixtures
   config.fixture_path = "#{::Rails.root}/spec/fixtures"
+  config.include Capybara::DSL
```
Comment (evettetelyas) [inline spec/rails_helper.rb:39]: check if you guys have this - i needed to add it to run some specs in capybara but who knows 🤷‍♀

File: app/models/user.rb [hunk 2]
```diff
@@ -26,6 +26,6 @@ def generate_token(column)
   end
 
   def works_here?(id)
-    @current_user.merchant.id == id
+    self.merchant.id == id
```
Comment (evettetelyas) [inline app/models/user.rb:29]: i tested this pretty well but maybe someone just check me that this is right. its the user model so self should be the same as current user

File: app/views/partials/_merchant_dash.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<center>
```
Comment (evettetelyas) [inline app/views/partials/_merchant_dash.html.erb:1]: paaaarrtttiiiaaalllssss 🙌

**Comment by tylorschafer** [comment]
Fingers crossed

**Pull request merged**